src/yb/tools/yb-admin_client.cc

```diff
@@ -1741,7 +1741,27 @@ Status ClusterAdminClient::ModifyPlacementInfo(
   // Iterate over the placement blocks of the placementInfo structure.
   std::unordered_map<std::string, int> placement_to_min_replicas;
   for (int iter = 0; iter < placement_info_split.size(); iter++) {
-    placement_to_min_replicas[placement_info_split[iter]]++;
+    std::vector<std::string> placement_info_min_replica_split = strings::Split(
+        placement_info_split[iter], ":", strings::AllowEmpty());
+
+    if(placement_info_min_replica_split.size() == 0 ||
+        placement_info_min_replica_split.size() > 2) {
+      return STATUS(InvalidCommand, "Each placement info must have at most 2 values separated by a colon. "
+          "Format: cloud.region.zone:[min_replica_count]. Invalid placement info: " + placement_info_split[iter]);
+    }
+
+    std::string placement_target = placement_info_min_replica_split[0],
+        placement_min_replica_count = "1";
+
+    if(placement_to_min_replicas.find(placement_target) != placement_to_min_replicas.end()) {
```

Comment:
In case a cloud.region.zone is mentioned twice, returing invalid command. Alternatives can be:
1. Final count can be total of all mentioned entries.
2. Use the first or last mentioned entry.